app/Http/Controllers/MerchantController.php

```diff
@@ -53,4 +53,17 @@ public function updateConfirm($id)
     
     return redirect('/merchantconfirmed');
   }
+
+  public function ordermasuk()
+  {
+     $user = User::with('transactions')->find(Auth::user()->id);
+     $users = $user->id;
+                                         
+                                      //  dd($users);  
+     $ordered = DB::Table('transactions')->where('status', 'accepted')
```

Comment:
why don't you use the model directly?
instead of using DB class, just use the model to keep it consistent